server/src/main/java/org/candlepin/liquibase/FixDuplicatePoolsTask.java

```diff
@@ -156,10 +149,11 @@ private int updateEntsForIds(List<String> dupeIds, String goodId) throws Excepti
             }
             else if (!subOwners.get(current).equals(rs.getString(4))) {
                 // Make sure owners are the same
-                log.error("Owners '{}' and '{}' both have pools from subscription: {}",
+                logger.error("Owners '{}' and '{}' both have pools from subscription: {}",
```

Comment:
Does this work? I thought the liquibase logger didn't support this style of interpolation?

Reply:
Just tested it, it does not.

```
this.logger.info("THIS IS A {}", "test");
```

Outputs:

```
20150210094558-35::crog: THIS IS A {}
```

Reply:
I replaced it with `String.format`